cobalt/umbrel-app.yml

```
@@ -0,0 +1,36 @@
+manifestVersion: 1
+id: cobalt
+name: cobalt
+tagline: A media downloader that doesn't piss you off
```

Use the tagline from their repo.
```suggestion
tagline: Best way to save what you love
```

---

I knew it wouldn't go through! 😂

---

ahaha